discordbot/stocks/screener/financial.py

```diff
@@ -0,0 +1,155 @@
+import difflib
+import discord
+import pandas as pd
+
+import discordbot.config_discordbot as cfg
+from discordbot.helpers import pagination
+
+from gamestonk_terminal.stocks.screener.finviz_model import get_screener_data
+
+
+async def financial_command(
+    ctx, preset="template", sort="", signal="", limit="25", ascend="False"
+):
+    """Displays returned results from preset by financial metrics [Finviz]"""
+    try:
+
+        # Debug
+        if cfg.DEBUG:
+            print(f"!stocks.scr.financial {preset} {sort} {signal} {limit} {ascend}")
+
+        # Check for argument
+        if not limit.lstrip("-").isnumeric():
+            raise Exception("Number has to be an integer")
+
+        limit = int(limit)
+
+        if limit < 0:
+            raise Exception("Number has to be above 0")
+
+        if ascend.lower() == "false":
+            ascend = False
+        elif ascend.lower() == "true":
+            ascend = True
+        else:
+            raise Exception("ascend argument has to be true or false")
+
+        # Output Data
+        df_screen = get_screener_data(
+            preset_loaded=preset,
+            data_type="overview",
+            limit=limit,
+            ascend=ascend,
+            signal=signal,
```

Comment:
the tests are failing because this signal keyword doesn't exist anymore. For all of them :)

Reply:
I'm already fixing it!

Reply:
Eh after trying to fix this and testing the changes, I've realized that it doesn't work without the keyword and I only get errors.